projects/1.20/assets/medieval-seedbags/seedbags/lang/zh_cn.json

```
@@ -0,0 +1,26 @@
+{
+  "block.seedbags.bag_of_melon_seeds": "西瓜种子袋",
+  "block.seedbags.pouch_of_melon_seeds": "西瓜种子包",
+  "block.seedbags.pouch_of_berrygood_glow_berry_pips": "发光浆果籽包",
+  "itemGroup.tabseedbags": "中世纪种子袋",
+  "block.seedbags.pouch_of_farmersdelight_tomato_seeds": "番茄种子包",
+  "block.seedbags.bag_of_beetroot_seeds": "甜菜根种子袋",
+  "block.seedbags.bag_of_pumpkin_seeds": "南瓜种子袋",
+  "block.seedbags.bag_of_wheat_seeds": "小麦种子袋",
+  "item_group.seedbags.seedbags": "中世纪种子袋",
+  "block.seedbags.pouch_of_pumpkin_seeds": "南瓜种子包",
+  "block.seedbags.bag_of_farmersdelight_tomato_seeds": "番茄种子袋",
+  "block.seedbags.bag_of_berrygood_sweet_berry_pips": "甜浆果籽袋",
+  "block.seedbags.bag_of_berrygood_foulberry_pips": "污秽浆果种子袋",
```

```suggestion
  "block.seedbags.bag_of_berrygood_foulberry_pips": "秽莓种子袋",
```
来自Autumnity，Team Abnormals系列作品

---

谢谢，已采纳